backend/inspirehep/mailing/cli.py

```diff
@@ -41,3 +47,34 @@ def send_weekly_jobs(test_emails):
     send_jobs_weekly_campaign(content, test_emails=test_emails)
 
     click.secho("Campaign sent.", fg="green")
+
+
+@mailing.command(help="Sends an email to the job's author for jobs which deadline is today or expired 30/60 days ago.")
+@click.option(
+    "-se", "--skip-emails", "skip_emails", help="Skip email sending.", is_flag=True
+)
+def notify_expired_jobs(skip_emails):
+    jobs_to_notify = []
+    dates = [
+        str(datetime.date.today()),
```

Comment:
```suggestion
        datetime.date.today().isoformat(),
```
makes it more clear how exactly it's formatted.